docs/articles_en/get-started/install-openvino/install-openvino-pip.rst

```diff
@@ -15,6 +15,19 @@ Install Intel® Distribution of OpenVINO™ Toolkit from PyPI Repository
    * is dedicated to users of all major OSes: Windows, Linux, and macOS
      (all x86_64 / arm64 architectures)
    * macOS offers support only for CPU inference
+   
+| **Simplified Build and Integration**  
+|   The package includes CMake configurations, precompiled static libraries, and headers, which
+    can be easily accessed through the Python API. You can use the `get_cmake_path()` method to
+    retrieve the paths to the CMake configurations and libraries:
+
+.. code-block:: python
+
+   from openvino import get_cmake_path
+   cmake_path = get_cmake_path()
+   
+For detailed instructions on how to use these configurations in your build setup, check out the
```

Comment:
maybe we can also show how to extract path to OpenVINO Config via a cmd line `python -c` and mention that it can be passed as cmake -DOpenvVINO_DIR=$(python -c "...")

Reply:
added here https://github.com/openvinotoolkit/openvino/pull/26695/files#diff-ab76de60539cb30e9b9548ca5578d723afc4b1df2a6d6189676f0fb32e4224e7